src/client/errors.ts

```diff
@@ -2,32 +2,89 @@
  * @module Errors
  */
 
+/**
+  * Error codes returned by actions involving creating applications.
+  */
 export enum OpenError {
+    /**
+     * Indicates that an application is unable to be found in the AppDir.
+     */
     AppNotFound = 'AppNotFound',
+    /**
+     * Indicates that an application cannot be started from an OF manifest.
+     */
     ErrorOnLaunch = 'ErrorOnLaunch',
+    /**
+     * Indicates that an application was not created in a timely manner, and the provider timed out.
+     */
     AppTimeout = 'AppTimeout',
+    /**
+     * Currently unused.
+     */
     ResolverUnavailable = 'ResolverUnavailable',
 }
 
+/**
+ * Error codes returned by attempts firing intents to applications.
```

Comment:
"attempts at" instead? "Error codes returned when attempting to fire an intents at an applications."?

Reply:
I cleaned this up